docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
+---------------------------
+
+Fixes and clean up:
+
+* clean up of bundles, packages and build targets to reflect decoupling work
+* POM repositories clean up
+* updated developer documentation on building Bio-Formats
+* updated testing packages
+* updated build versioning from Maven
+
+Further decoupling:
+
+* removed stubs and metakit and replaced them with jars from Maven Central
```

Comment:
As we did previously, we might want to add extra information about the repository and `groupId/artifactId`